JavaApplication1/src/javaapplication1/Cuenta.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package javaapplication1;
+
+/**
+ *
+ * @author Jorge
+ */
+public class Cuenta {
+
+    public String titular;
+    public Double cantidad;
+    
+    
+    public Cuenta(String getTitular) {
+        this.titular = getTitular;
+        this.cantidad = 0.00;
+    }
+    
+    public Cuenta(String getTitular, Double getCant) {
+        this.titular = getTitular;
+        this.cantidad = getCant;
+    }
+
+    public String getTitular() {
+        return titular;
+    }
+
+    public void setTitular(String tit) {
+        this.titular = tit;
+    }
+    
+    public String toStringTitular() {
+        return "Accede o setea el nombre del titular de la cuenta";
+    }
+    
+    public Double getCantidad() {
+        return cantidad;
+    }
+
+    public void setCantidad(Double cant) {
+        this.cantidad = cant;
+    }
+    
+    public String toStringCantidad() {
+        return "Accede o setea la cantidad de dinero de la cuenta";
+    }
+    
+    public void ingresar(Double cant) {
+        if(cant > 0){
+            this.cantidad += cant;
+        }
+    }
+    
+    public void retirar(Double cant) {
+        
+        if(cant > this.cantidad){
```

Comment:
Esto esta mal planteado , pues to que plantea que solo se podrá retirar dinero si la cantidad a retirar es mayor a la cantidad existente, debería de plantearse de la siguiente manera:
Si la cantidad a retirar menos la cantidad existente no es menor que 0.

Reply:
@pujolman